app/views/admin/enterprises/form/_business_details.html.haml

```diff
@@ -57,3 +57,17 @@
   .row{"data-controller": "updateinput"}
     = bf.hidden_field :_destroy, {"data-updateinput-target": "input"}
     = f.submit t(".reset_form"), {class: 'secondary', "data-action": "click->updateinput#update", "data-updateinput-value": "true"}
+
+.row
+  %fieldset.alpha.no-border-bottom
+    %legend= t('Invoice_item_sorting')
```

Comment:
Why not the standard way?
```suggestion
    %legend= t('.invoice_item_sorting')
```

I see, you followed the existing pattern. That's okay but ideally we would move everything to the `.key` version.